imports/assemblyscript/assemblyscript.go

```diff
@@ -272,6 +272,7 @@ var seed = &wasm.HostFunc{
 	ExportNames: []string{functionSeed},
 	Name:        "~lib/builtins/seed",
 	ResultTypes: []api.ValueType{f64},
+	ResultNames: []string{"rand"},
```

Comment:
I put result names for anything somewhat interesting. For example, the function name here is `seed` but it might not be intuitive that the seed value is a random number (some use clocks etc).